kong/api/routes/clustering.lua

```diff
@@ -1,18 +1,23 @@
-local clustering = require("kong.clustering")
-local kong = kong
+local endpoints = require "kong.api.endpoints"
+
+
+local dp_collection_endpoint = endpoints.get_collection_endpoint(kong.db.clustering_data_planes.schema)
 
 
 return {
-  ["/clustering/status"] = {
-    GET = function(self, db, helpers)
-      if kong.configuration.role ~= "control_plane" then
-        return kong.response.exit(400, {
-          message = "this endpoint is only available when Kong is " ..
-                    "configured to run as Control Plane for the cluster"
-        })
-      end
+  ["/clustering/data_planes"] = {
```

Comment:
Should we call this `data_nodes` instead?
`data_planes` is wrong grammar at this point.

cc @hishamhm